evap/staff/views.py

```diff
@@ -809,6 +815,10 @@ def course_comments_update_publish(request):
 def course_comment_edit(request, semester_id, course_id, text_answer_id):
     semester = get_object_or_404(Semester, id=semester_id)
     course = get_object_or_404(Course, id=course_id, semester=semester)
+
+    if not course.has_enough_voters_to_publish_text_answers:
+        raise PermissionDenied
```

Comment:
if i understand correctly, you want to prevent *everyone*, even staff users, to see these comments. in that case, we should probably delete them on publish.